src/js/server/hcapi/applications.js

```diff
@@ -4,6 +4,23 @@ const { createHttpError } = require('./errors');
 
 const applicationsRouter = new Router();
 
+applicationsRouter.get('/', (req, res, next) => {
+  HackerApplication
```

Comment:
I'm still not sure getting everything by application is the best plan, especially as we already have a way of [computing application status on the Hacker model](https://github.com/hackcambridge/hack-cambridge-website/blob/master/src/js/server/models/Hacker.js#L40)

That being said, maybe we should just get this out the door. Keen to hear your thoughts (and possible @jaredkhan's too)